sdk/monitor/monitor-opentelemetry/src/metrics/quickpulse/filtering/validator.ts

```diff
@@ -60,6 +59,15 @@ export class Validator {
     });
   }
 
+  public validateDocumentFilters(
+    documentFilterConjuctionGroupInfo: DocumentFilterConjunctionGroupInfo,
+  ): void {
+    documentFilterConjuctionGroupInfo.filters.filters.forEach((filter) => {
```

Comment:
It would be better to separate the FilterConjunctionGroupInfo field(filters) as a variable here and then iterate through the internal filters, this to improve readability and make it clear that is not some typo or something

Reply:
That's fair, it does look confusing the way it is. Will fix